static/app/views/issueDetails/streamline/eventList.tsx

```diff
@@ -170,10 +171,10 @@ export function EventList({group}: EventListProps) {
 
 const EventListHeader = styled('div')`
   display: grid;
-  grid-template-columns: 1fr auto auto auto;
+  grid-template-columns: 1fr auto auto;
   gap: ${space(1.5)};
   align-items: center;
-  padding: ${space(1)} ${space(2)};
+  padding: ${space(1)} ${space(1)} ${space(1)} ${space(2)};
```

Comment:
i _think_ we wanted all events, event id, and event to all be aligned, not just the bottom two

Reply:
![image](https://github.com/user-attachments/assets/4bc340cb-ca20-488d-84e6-0c399ddae3fa)

after changes